frontend/src/scenes/error-tracking/ErrorTrackingScene.tsx

```diff
@@ -92,27 +92,13 @@ export function ErrorTrackingScene(): JSX.Element {
 }
 
 const VolumeColumn: QueryContextColumnComponent = (props) => {
-    const { sparklineSelectedPeriod, customSparklineConfig } = useValues(errorTrackingLogic)
     const record = props.record as ErrorTrackingIssue
-
-    if (!record.aggregations) {
-        return null
-    }
-
-    const [data, labels] =
-        sparklineSelectedPeriod === '24h'
-            ? [record.aggregations.volumeDay, sparklineLabelsDay]
-            : sparklineSelectedPeriod === '30d'
-            ? [record.aggregations.volumeMonth, sparklineLabelsMonth]
-            : customSparklineConfig
-            ? [record.aggregations.customVolume, sparklineLabels(customSparklineConfig)]
-            : [null, null]
-
-    return data ? (
+    const [values, unit, interval]: [number[], TimeUnit, number] = useSparklineData(record.aggregations)
```

Comment:
We shouldn't need the extra typing here, right?
```suggestion
    const [values, unit, interval] = useSparklineData(record.aggregations)
```